modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/processors/OperationProcessor.java

```diff
@@ -70,8 +70,8 @@ public void processOperation(Operation operation) {
             for (String responseCode : responses.keySet()) {
                 ApiResponse response = responses.get(responseCode);
                 if(response != null) {
-                    //This part allows parser to put response inline without the resolveFully option set to true
-                    if (response.get$ref() != null) {
+                    //This part allows parser to put response inline within resolveResponses or ResolveFully
```

Comment:
would replace with `// This part allows parser to put response inline when resolveResponses = true`

see below